Manuscript/Text/02F_mutein.md

```diff
@@ -1,3 +1,9 @@
+
+It is desirable for many clinical applications for a drug to induce a purely immunosuppresive response. In order to quantify the effectiveness of these modified and wild-type cytokines in selectively activating immunosuppressive T~reg~ cells, we defined formulated specificity functions. These functions were defined as pSTAT5 activity of T~reg~s divided by pSTAT5 activity of T-helper cells or NK cells. Specificty of T~reg~ activation with respect to NK and T-helper cells was experimentally determined and predicted by our model to be largest at low concentrations of IL-2, but was not found to have a significant relationship with IL-15 concentration ([@fig:mutFac]B/C). Because high T~reg~ specificity is desirable for several clinical applications of cytokine-based drugs, we also quantified the partial derivatives of our specificity functions with respect to both surface and endosomal binding constants, and found that increasing IL-2Rα affinity while decreasing IL-2Rβ/γ~c~ affinity on the cells surface would most drastically increase the cytokine's T~reg~ activation specificity ([@fig:mutFac]D). These effects were notably different between surface and endosomal binding rates. Furthermore, our model predicts that ligands with reduced IL-2Rα affinity had a decreased ability to specifically activate T~reg~s with respect to NK and T-Helper cells over any degree of IL-2Rβ/γ~c~ affinity reduction ([@fig:mutFac]E). (put something specifically about current industry drug-discovery/design here?) Our model was also able to accurately guage the effect that changes in ligand-receptor binding affinities had on their respective EC~50~s ([@fig:mutFac]F). Ligands with decreased IL-2Rα affinity had increased EC~50~s for T~reg~ varieties, and ligands with decreased IL-2Rβ/γ~c~ affinities had larger EC~50~s for T-Helper cells, in accordance with our previous findings.
```

Comment:
You can start more direct. Say, "We sought to identify strategies for selectively targeting T~reg~s"

Reply:
Rather than general statement about surface vs. endosome, how about: "Interestingly, specificity was more sensitive to endosomal IL-2Rα affinity." You could mention this paper as previous work on the idea that engineering endosomal affinity can bring beneficial properties: PMID: 12161759